### ISSUE TITLE
Add valid PHPDoc

### ISSUE THREAD

**Issue by ruudk** [itavero/AMNL-Mollie#16]
For instance

``` php
    /**
     * Returns an array of Bank-instances the customer can choose from.
     *
     * @param boolean $use_cache Use internal array cache?
     * @return array An array containing Bank-instances
     */
    public function getBankList($use_cache = true)
```

Should be

``` php
    /**
     * Returns an array of Bank-instances the customer can choose from.
     *
     * @param boolean $use_cache Use internal array cache?
     * @return \AMNL\Mollie\IDeal\Bank[]
     */
    public function getBankList($use_cache = true)
```

That way, an IDE like PHPStorm, can do better autocompletion.

**Comment by itavero** [comment]
As far as I know the IDE I use does not use this, so that's probably why I've typed array instead.
Please let me know if there are more PHPDoc issues like this. I'll try to look over the PHPDoc soon.

**Comment by ruudk** [comment]
You should test it out in PHPStorm (great IDE for Mac).

**Comment by ruudk** [comment]
Partially fixed by #18
